app/create.js

```diff
@@ -76,17 +83,34 @@ define(['app/lang', 'app/string', 'app/config', 'app/database', 'jquery', 'boots
     function buildDatasetSubjectString(form, select, language)  {
       var subject = forms.getFormVal(form, select, subject);
       return subject;
-    }    
+    }
 		// Script when the form is successful
 		forms.initializeForm('#createForm', function() {
 			showLoading(function() {
 				var form = "#createForm";
-				forms.saveDataset(form);
-				var id = db.lastInsertRowId("tbldatasets", "dataset_id");
-        forms.getItemsFromCreateForm(id, formItemId);
-				var language = buildDatasetLanguageString(form, "select", "language");
-	      var subject = buildDatasetSubjectString;
-				window.location = "index.html?message=create_dataset&language="+language+"&subject="+subject;
+				var name = forms.getFormVal(form, "input", "name");
+	      var language = forms.getFormVal(form, "select", "language");
+	      var subject = forms.getFormVal(form, "select", "subject");
+	      var user_id = user.getCookie('user_id');
+	      var currentdate = date.formatDatetime(new Date(), true);
+
+				var dataset_items = [];
```

Comment:
It looked to me as if the left margin jumped strangely. But based on your response I'd say it is due to GitHub. If you don't think it is formatted strangely, I believe you. 

Sent from my phone

> On 31 May 2016, at 13:53, Enrico Werkema <notifications@github.com> wrote:
> 
> In app/create.js:
> 
> >  		// Script when the form is successful
> >  		forms.initializeForm('#createForm', function() {
> >  			showLoading(function() {
> >  				var form = "#createForm";
> > -				forms.saveDataset(form);
> > -				var id = db.lastInsertRowId("tbldatasets", "dataset_id");
> > -        forms.getItemsFromCreateForm(id, formItemId);
> > -				var language = buildDatasetLanguageString(form, "select", "language");
> > -	      var subject = buildDatasetSubjectString;
> > -				window.location = "index.html?message=create_dataset&language="+language+"&subject="+subject;
> > +				var name = forms.getFormVal(form, "input", "name");
> > +	      var language = forms.getFormVal(form, "select", "language");
> > +	      var subject = forms.getFormVal(form, "select", "subject");
> > +	      var user_id = user.getCookie('user_id');
> > +	      var currentdate = date.formatDatetime(new Date(), true);
> > +
> > +				var dataset_items = [];
> How would you like to see this formatted?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>